common/djangoapps/student/views.py

```diff
@@ -1599,6 +1616,22 @@ def create_account_with_params(request, params):
         getattr(settings, 'REGISTRATION_EXTRA_FIELDS', {})
     )
 
+    # Start: Added by Labster
+    # User can create an account only on the appropriate region or user
+    # that already enrolled in this current region.
+    email_enrolled = CourseEnrollmentAllowed.objects.filter(email=params["email"]).exists()
+    if not email_enrolled:
+        # If the user region is covered by Labster, redirect student to the appropriate region server.
+        current_region = request.session.get('country_code')
+        base_url = request.build_absolute_uri('/')[:-1]
+        regions = configuration_helpers.get_value('REGIONS', settings.REGIONS)
+        for region_code, region in regions.items():
+            register_url = region['register_url']
+            if current_region == region_code and base_url not in register_url:
+                return redirect(register_url)
```

Comment:
I'm afraid it will introduce much more question than help.
Why?

as I understand it works this way:
I'm a student from UK
And I open Central edX
Go to sign up page
Fill the data
Select `Sign UP` button
Then page reloads and I see the same form but it is empty.

I even can not suspect that I was redirected to another one site (from Central region to UK), so I will write to CS and ask why I cannot create my account. 
@avkoval @lukyfebrianto have you tested this flow? is it useful?

Reply:
yes it'll make some user confused.
@polesye I think we need some message like previous changes instead of redirect to the region server.

Reply:
@lukyfebrianto I agree, but it's better to ask @avkoval.

Reply:
Yes @lukyfebrianto , it is explained in my diagram as "display redirection message". Please contact Emma to obtain the text I've already asked her about it.

In this case it seems we should redirect to the other edX URL plus some view which will render this redirection message for this purpose.